docs/source/conf.py

```diff
@@ -29,10 +31,22 @@
 
 # -- Options for HTML output
 
+# select theme from options below
 html_theme = 'sphinx_rtd_theme'
+#html_theme = 'sphinx_book_theme'
```

Comment:
Should this be commented out?

Reply:
yeah, was meant to be an alternate choice of theme